float64 pos_x
float64 pos_y
---
int64 x
int64 y
